spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -19,42 +19,34 @@
         it_behaves_like 'successful request', :index
       end
 
-      context 'with @category set' do
-        it 'populates an array of of active category-type equipment models' do
+      it 'defaults to all active equipment models' do
+        models = mock_eq_models
+        stub_collection_methods models
+        allow(EquipmentModel).to receive(:where).and_return([])
+        expect_any_instance_of(Array).to \
+          receive_message_chain(:includes, :active)
+        get :index
+      end
+
+      context '@category set' do
```

Comment:
Okay, here we don't actually need (or want) to `stub_collection_methods`, which I think is the main issue. The stubs are set up differently here (which is hidden away in the helper), so I think the issue is that they need to be more explicit